social_auth.install

```diff
@@ -28,14 +45,14 @@ function social_auth_update_8001(&$sandbox) {
  * key post_login_path to post_login.
  */
 function social_auth_update_8002(&$sandbox) {
-  $config = \Drupal::configFactory()->getEditable('social_auth.settings');
-  $post_login = $config->get('post_login_path');
-
-  if ($post_login == 'user') {
-    $config->set('post_login', '/user');
-  }
-  else {
-    $config->set('post_login', $post_login);
-  }
-  $config->save();
+    $config = \Drupal::configFactory()->getEditable('social_auth.settings');
+    $post_login = $config->get('post_login_path');
+
+    if ($post_login == 'user') {
+        $config->set('post_login', '/user');
+    }
+    else {
+        $config->set('post_login', $post_login);
+    }
+    $config->save();
```

Comment:
Why were this lines indented with 4 spaces? The standard is 2 spaces. You can use [PHP CodeSniffer](https://www.drupal.org/project/coder) to check for this kind of things.